docs/DeveloperGuide.md

```diff
@@ -288,7 +290,7 @@ For the 4 classes listed, there were some other design considerations that was d
     * Cons:
         - Lower SRP and (SoC)
 
-###### 2.2.4.5.1 Aspect: Autosaving or no
+###### 2.2.4.5.2 Aspect: Autosaving or no
```

Comment:
Consider another way of labeling  sub-sections as using 2.2.4.5.1 etc can be quite hard to follow and tedious to edit as well.